sync/class.jetpack-sync-listener.php

```diff
@@ -259,28 +259,29 @@ function get_actor( $current_filter, $args ) {
 			'is_wp_admin'      => is_admin(),
 		);
 
-		if ( $this->should_send_ip_address_with_actor( $current_filter ) ) {
+		if ( $this->should_send_user_data_with_actor( $current_filter ) ) {
 			require_once( JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php' );
 			$actor['ip'] = jetpack_protect_get_ip();
+			$actor['user_agent'] = $_SERVER['user_agent'];
```

Comment:
I think we should check the presence of this user_agent. 

also what user agent is set when the request comes from an API?

Reply:
Good call Enej. it looks like user_agent is not present during an API request.

For now, i'll set it to 'unknown', but we could certainly get fancier with it in future iterations.